File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/fab/FabActivity.java
package com.jdroid.android.sample.ui.fab;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class FabActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return FabFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/fab/FabActivity.kt
package com.jdroid.android.sample.ui.fab

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class FabActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return FabFragment::class.java
    }
}
